### ISSUE TITLE
feature request: box.com sync support

### ISSUE THREAD

**Issue by kribjo** [PhilippC/keepass2android#285]
I would like to see support for syncing with box.com.

Some details on this is available on the trigger examples page https://keepass.info/help/kb/trigger_examples.html#syncbox

**Comment by Offerel** [comment]
Does box.com not have WebDAV Support via https://dav.box.com/dav ?

**Comment by oneoftwo** [comment]
> Since April 30, 2018 box.com terminated webdav support. (https://community.box.com/t5/Box-Product-News/Deprecation-WebDAV-Support/ba-p/55684)
> I would like to see support for box.com too...

Seconded. It would be great if box.com were natively supported, seeing as dav.box.com/dav will go away in just two months. Thanks!

**Comment by PhilippC** [comment]
please use the thums-up-reaction instead of writing "me too" comments. Thanks.

**Comment by flo-at** [comment]
as of today, WebDav is still working on Box.com. But that might change at any time.

**Comment by Ietek** [comment]
Hi PhilippC, Do you have any news about box.com sync support? Thanks.

**Comment by PhilippC** [comment]
no, sorry. I have been working on some upgrades, but the requests come in faster than I can close them :-(

**Comment by viz6411** [comment]
@PhilippC Hello! Due to recent events, I would like to know if it is worth waiting for the addition of native support for box.com ? Thanks.

Box completely disabled support for WebDav and now there is no way to synchronize the database with this cloud. 
https://support.box.com/hc/en-us/articles/360052806073-Announcing-end-of-life-for-Box-WebDAV-support

**Comment by Gian-Fr** [comment]
I suggest to put the vote-thumb-up-if-you-like-it label and let users decide if this is relevant
